_i18n/en.yml

```diff
@@ -1021,7 +1020,6 @@ tools:
 
   network: Network
   monerohash-nodes: Monero node map
-  localmonero-blocks: Explorer and Statistics
```

Comment:
if the blockchain explorer will continue this line should stay.

Reply:
it's a block explorer, this PR moves it under block explorers. a mirror of the "moneroblocks" that was listed under block explorers. it is now redundant text